### ISSUE TITLE
Feature Request: Automatic back-and-forth when switching workspaces

### ISSUE THREAD

**Issue by aengelas** [leftwm/leftwm#65]
As a long-time i3 user, one of my favorite features is the automatic back and forth support they have when switching workspaces (https://i3wm.org/docs/userguide.html#workspace_auto_back_and_forth).

If you switch to a workspace, and then try to switch to the same workspace again, you instead jump back to the workspace you came from. I find this very handy.

This looks like an awesome project. I LOVE the support for multiple workspaces per monitor, etc.

**Comment by lex148** [comment]
So there is no way currently of binding this to the active desktop, but are
(mod + w) and (mod + shfit + w) kind of what you are thinking about? 
or are you thinking it should be a bit different?

**Comment by aengelas** [comment]
I'm not sure exactly what mod+w and mod+shift+w do, but I'm thinking that it would look like this (a series of actions taken by the user):

```
mod+1 # focus tag 1 (if my understanding of leftwm is correct)
mod+2 # focus tag 2
mod+2 # focus tag 1
```

Does that make sense? I'm not sure how to best explain it. It's hard to articulate. The description from the i3 link above is probably the clearest articulation.

**Comment by murlakatamenka** [comment]
> I'm not sure how to best explain it.

Let me try: it's like Alt + Tab, but for switching workspaces, not windows, and just 2 latest, not all. I have `i3`'s back-and-forth mapped to Alt + Tab exactly. Sounds good?

**Comment by PVautour** [comment]
@aengelas I have this working on a fork of mine. I just need to tidy it up and i will submit a pull request :)

**Comment by PVautour** [comment]
@lex148 I believe we can mark this as resolved and close the issue. It is resolved by #208 .

PS: Thanks for all the good work you do you are really putting the effort on managing the project and it shows! Keep it going and we will have the perfect window manager in no time!
